OrbitGgp/InstanceTests.cpp

```diff
@@ -14,37 +14,20 @@ namespace OrbitGgp {
 
 TEST(InstanceTests, GetListFromJson) {
   QByteArray json;
-  QVector<Instance> test_instances;
-  Instance test_instance;
 
   // invalid json
   json = QString("json").toUtf8();
-  test_instances = Instance::GetListFromJson(json);
-  EXPECT_TRUE(test_instances.empty());
+  EXPECT_FALSE(Instance::GetListFromJson(json));
```

Comment:
The comments introduce subsections, right?
Maybe put them into curly braces, such that every variable's scope is limited where it is actually needed and no data accidentally leaks into the next sub-test.

Reply:
good idea, done